skype/skype-ps/skype/skype.md

```diff
@@ -1009,6 +1012,9 @@ The following cmdlet references are for Skype for Business and Microsoft Teams.
 ### [Grant-CsTeamsMeetingPolicy](Grant-CsTeamsMeetingPolicy.md)
 {{Manually Enter Grant-CsTeamsMeetingPolicy Description Here}}
 
+### [Grant-CsTeamsMeetingBrandingPolicy](Grant-CsTeamsMeetingBrandingPolicy.md)
+{{Manually Enter Grant-CsTeamsMeetingBrandingPolicy Description Here}}
```

Comment:
You need to copy the sentence from the SYNOPSIS to replace the placeholder text

Reply:
@sdwheeler I followed how to existing commands are in this file, all of them have this {{Manually Enter ... }} template.

When I open the preview, it seems that description is correctly filled with SYNOPSIS from the document, so what's the point of copying it?

https://review.learn.microsoft.com/en-us/powershell/module/skype/?view=skype-ps&branch=pr-en-us-10657
![image](https://user-images.githubusercontent.com/128838585/229742868-e4bc8f8f-ec69-4cc4-85b3-a44d5ddf9746.png)

Reply:
The build system is not choosing the correct text when you don't replace the placeholder. This should be a single sentence description of the cmdlet.

Reply:
Done, copied short cmdlet description to skype.md